atom/browser/api/lib/web-contents.js

```diff
@@ -106,16 +112,37 @@ let wrapWebContents = function(webContents) {
     };
   }
 
+  for (let method of asyncWebFrameMethods) {
+    webContents[method] = function() {
+      let args = Array.prototype.slice.call(arguments);
```

Comment:
We can use ES6 rest arguments:
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/rest_parameters